### DISCUSSION TITLE
command "model_devi_activation_func" problem

### DISCUSSION THREAD

**Discussion by WhiteCrosstheRiver** [deepmodeling/dpgen#941] [Q&A] [answered]
Here,my param.json and machine.json are already finished init_bulk process,and turned into stege4 --successfully collected data--
However,when I USE param2.json and machine.json to execute the command "dpgen run param.json machine.json" with
"numb_models": 4,
It always with [["tanh","tanh"],["tanh","gelu"],["gelu","tanh"],["gelu","gelu"]] these 4 [combinations] in each initial dp training documents 000 001 002 003 no matter how I change command "model_devi_activation_func" ...

[my_script_and_someoutputs.zip](https://github.com/deepmodeling/dpgen/files/9539661/my_script_and_someoutputs.zip)

@njzjz DEAR Dr.Zeng,Have you ever notied some kind of like this situation?If so,many thanks to you tell me later!!!

**Comment by njzjz** [comment, marked as answer]
`dpgen.log` shows `continue from iter 000 task 00`, which means task 0 has been completed. You have to revert `record.dpgen` to redo task 0.

**Reply by WhiteCrosstheRiver** [reply]
THANKYOU!!!Dr.Zeng
Factly，I was going the example https://zhuanlan.zhihu.com/p/555628454 & https://zhuanlan.zhihu.com/p/555544567 ...

I have redone it over and over again :
Howver it always gives  me the error --LookupError: gradient registry has no entry for: Gelu-- when fiting net OR embbeding activation function  is set 'gelu'  meanwhile   it can be trained in GPU when fiting net AND embbeding activation function  is set 'tanh'

**Reply by njzjz** [reply]
@denghuilu

**Reply by denghuilu** [reply]
https://github.com/deepmodeling/deepmd-kit/pull/1918 should fix this issue.

**Reply by WhiteCrosstheRiver** [reply]
TOO much thanks for you two!!!From NanJing!!!
this little bug has been fixed since installed the version 2.1.5 of deepmdkit
however,I dont know why running the command (dpgen run param machine)
the remote work file dirs' deepmd shows something missing
![图片](https://user-images.githubusercontent.com/53998985/192297762-8a1a4f68-876b-4c57-ae2b-e2d4c64a6ebf.png)
well we can find after init_bulk running process,it will give a lot
![图片](https://user-images.githubusercontent.com/53998985/192298214-f9bff106-b11c-495a-acd6-71890d5c0f00.png)

for the question of losing something matter,the trainning running process always broken down
![图片](https://user-images.githubusercontent.com/53998985/192299768-d00702cf-0eae-450c-a798-52cd9fe961c8.png)

**Reply by WhiteCrosstheRiver** [reply]
And when I force copying items losing to the dirs init_data,it could train successfully but failed continuing traning 
![图片](https://user-images.githubusercontent.com/53998985/192300403-6418c066-475d-4a21-babf-c9872407e708.png)
